deis/monitor
# chore(grafana): Update dashboards with new graphs

- NSQ - Add cpu and memory usage, update color values for message topic depth and messages per second
- Deis Health - Aggregate CPU usage across all containers of a similar pod name
- Deis Router - Add CPU and Memory, update location of certain graphs
- Influx - Updated memory usage and disk usage to be single stat graphs
- Kubernetes Health - Add aggregate pod memory usage and api resource summary latency graphs
- Redis - Make memory a single stat panel

Testing:
- Have fresh cluster with latest redis changes
- `make build push upgrade` in grafana directory
- Login `grafana.mydomain.com` `admin/admin`
- Verify all dashboards are present (listed above) 
- Make sure values are flowing into the graphs
